modules/postgres/src/main/scala/doobie/postgres/syntax/syntax.scala

```diff
@@ -614,4 +618,73 @@ trait ToPostgresMonadErrorOps {
     new PostgresMonadErrorOps(ma)
 }
 
+trait ToPostgresExplainOps {
+  implicit def toPostgresQuery0Ops(q: Query0[_]): PostgresExplain0Ops =
+    new PostgresExplain0Ops(q)
+
+  implicit def toPostgresQueryOps[A](q: Query[A, _]): PostgresExplainOps[A] =
+    new PostgresExplainOps(q)
+}
+
+class PostgresExplain0Ops(self: Query0[_]) {
+  /**
+   * Construct a program in
+   * `[[doobie.free.connection.ConnectionIO ConnectionIO]]` which returns the server's query plan
+   * for the query (i.e., `EXPLAIN` output). The query is not actually executed.
+   */
+  def explain: ConnectionIO[List[String]] =
+    self.inspect { (sql, prepare) =>
+      HC.prepareStatement(s"EXPLAIN $sql")(for {
+        _ <- prepare
+        result <- HPS.executeQuery(HRS.build[List, String])
+      } yield result )
```

Comment:
Can you simplify the `for` comp to `prepare *>  HPS.executeQuery(HRS.build[List, String]` here and in the other instances?

Reply:
The `*>` method is unfamiliar to me. When I write `HC.prepareStatement(s"EXPLAIN $sql")(prepare *> HPS.executeQuery(HRS.build[List, String]))`, I get a compile error:

```
[error] \doobie\modules\postgres\src\main\scala\doobie\postgres\syntax\syntax.scala:638:52: value *> is not a member of doobie.PreparedStatementIO[Unit]
[error]       HC.prepareStatement(s"EXPLAIN $sql")(prepare *> HPS.executeQuery(HRS.build[List, String]))
```

Which import do I need to make it work?

Reply:
`cats.implicits._` and possibly `doobie.implicits._`

Reply:
Thank you! Changes pushed :)